src/docs/product/performance/metrics.mdx

```diff
@@ -88,6 +77,17 @@ Each of these functions is calculated with respect to the collection of transact
 
 ## User Misery
 
-User Misery is a user-weighted performance metric to assess the relative magnitude of your application performance. While you can examine the ratio of various response time threshold levels with [Apdex](#apdex), User Misery counts the number of unique users who were frustrated based on the specified response time threshold. User Misery highlights transactions that have the highest impact on users.
+User Misery is a user-weighted performance metric to assess the relative magnitude of your application performance. While you can examine the ratio of various response time threshold levels with [Apdex](#apdex), User Misery counts the number of unique users who were frustrated based on 4x the satisfactory response time threshold (ms). User Misery highlights transactions that have the highest impact on users.
+
+In our Early Adopter program, satisfactory thresholds can be set per project with [custom thresholds](#custom-thresholds).
+
+## Custom Thresholds
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+For each project, you can configure how [Apdex](#apdex) and [User Misery](#user-misery) are calculated in **[Project] > Settings > Performance**.
 
-If you're in the Early Adopter program, you can set response time [thresholds per Project](#set-thresholds-per-project).
+The calculation method determines if duration is defined as the entire length of the transaction or as a specific [Web Vital](/product/performance/web-vitals/) such as LCP. The response time threshold determines what the satisfactory baseline duration is in milliseconds. This may vary across projects depending how user-facing it is.
```

Comment:
@doralchan I'm not sure what "this" and "it" are referring to in the last sentence. Is "this" the threshold or the baseline? Please add that word in whichever it should be. And is "it" in the last sentence the project or the baseline? I've edited it to be the project, but if it's not, please edit to explicitly say it's the baseline there. We've got a lot of nouns in those last two sentences (threshold, baseline, project), so it gets a bit unclear. Happy to review again when you've made changes if you'd like.

```suggestion
The calculation method determines if duration is defined as the entire length of the transaction or as a specific [Web Vital](/product/performance/web-vitals/) such as LCP. The response time threshold determines what the satisfactory baseline duration is in milliseconds. This may vary across projects depending on how user-facing a project is.
```